### ISSUE TITLE
Make the Error object's ref property a "normal" ref

### ISSUE THREAD

**Issue by ComLock** [react-hook-form/react-hook-form#12658]
**Is your feature request related to a problem? Please describe.**
I'm wrapping input elements from a frontend framework that marks it's input element with aria-errormessage= useId();

When implementing an ErrorSummary component to show all form errors, I want to get the aria-errormessage attribute, but the error.ref isn't a real reference to the input element. It has a focus() method which is useful to "scroll" to the related input. But it does not have a getAttribute() method.

Adding a getAttribute() method would be nice, but a full normal ref would probably be even better?

**Describe the solution you'd like**
See above

**Describe alternatives you've considered**
Keeping a mapping of field names, field ids, and error ids and make it available via a context provider/consumer.

**Comment by ComLock** [comment]
The "aria-errormessage" attribute can only contain one element id, and the "id" global attribute must be unique in the whole document. So if an <ErrorMessage> is displayed twice, only one of them can be linked to the input field via aria-errormessage.

So I think I don't need this feature after all.
